### PR TITLE
ApiV4: GET /teams

### PR DESCRIPTION
**Pull request by andreistanciu24**

### CODE REVIEW

File: source/v4/teams.yaml [hunk 1]
```diff
@@ -40,6 +40,36 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+    '/teams':
+    get:
+      tags:
+        - teams
+      summary: Get a list of all teams.
```
Comment (jwilander) [inline source/v4/teams.yaml]: Can you change this to `Get teams`?

File: source/v4/teams.yaml [hunk 2]
```diff
@@ -40,6 +40,36 @@
...
+      description: |
+        Logged in, only shows open teams. Shows all teams with "manage_system" permission. The result is based on query string parameters - page and per_page.
```
Comment (jwilander) [inline source/v4/teams.yaml]: Can you change this to `For regular users only returns open teams. Users with the "manage_system" permission will return teams regardless of type. The result is based on query string parameters - page and per_page.`

File: source/v4/teams.yaml [hunk 3]
```diff
@@ -40,6 +40,36 @@
...
+        - name: per_page
+          in: query
+          description: The number of teams per page.
+          default: "60"
+          type: string
+      responses:
+        '200':
+          description: Team list retrieval successful
+          schema:
+            $ref: '#/definitions/Team'
```
Comment (jwilander) [inline source/v4/teams.yaml]: This needs to be an array, see other endpoints for examples

**Comment by andreistanciu24** [comment]
done.

**Pull request merged**